src/middlewared/middlewared/plugins/account.py

```diff
@@ -415,11 +419,18 @@ async def do_update(self, pk, data):
         if home_copy and not os.path.isdir(user['home']):
             try:
                 os.makedirs(user['home'])
-                await self.middleware.call('filesystem.chown', {
+                mode_to_set = user.get('home_mode')
+                if not mode_to_set:
+                    mode_to_set = oct(old_mode)[-3:] if old_mode else '700'
```

Comment:
The correct way is
```
>>> '%03o' % 493
'755'
```

Reply:
Not really, % is old formatting style we are avoiding

Reply:
```
>>> f'{493:03o}'
'755'
```